src/Countries/Myanmar.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+class Myanmar extends Country
+{
+    public function countryCode(): string
+    {
+        return 'mm';
+    }
+
+    protected function allHolidays(int $year): array
+    {
+        return [
+            'လွတ်လပ်ရေးနေ့' => '01-04',
```

Comment:
Are all the holidays in Myanmar on a fixed date?

Reply:
Yes Bro. Those all dates are fixed and there will be two variable dates which may change everyyear. So skipped those days.

Reply:
Data Check Link

https://evisa.moip.gov.mm/home/publicholiday

Reply:
Are those variable dates based on Islamic lunar holidays?

Reply:
i think, No bro. 

first day is Eid al-Fitr (https://en.wikipedia.org/wiki/Eid_al-Fitr)
second day is deepavali (https://www.almanac.com/content/diwali)

Should i change day name to English bro ?